### DISCUSSION TITLE
Sequences with MikroORM

### DISCUSSION THREAD

**Discussion by WillianSugiyama** [mikro-orm/mikro-orm#4734] [Q&A]
Hi, how i can make an entity with a sequence number based on sequence by postgresql?

Like, I have the entity user with:

```
@Entity()
export class Sheet {
  @PrimaryKey()
  id: string = v4();
  
  @Property();
  creationId: number;
}
```

Where creationId is a sequence from postgresql?

What is the best solution to solve this?

**Comment by B4nan** [comment]
You could do this:

```ts
  @Property({ type: 'serial' });
  creationId: number;
```

This should work as well:

```ts
  @Property({ autoincrement: true });
  creationId: number;
```
